_posts/2018-08-15-JS-SDK.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: The JS SDK hits version 2.0
+date: 2018-08-15 15:16 -0400
+comments: true
+author: flovilmart
+categories: [announcements, javascript, sdk]
+---
+
+The summer has been really busy and we've dusted out the JS SDK and striped large parts of it's heritage. `Parse.Promises` and `Backbone` style callbacks are definitely out.
```

Comment:
The summer has been really busy and we've dusted off the JS SDK, stripping out large parts of its heritage. `Parse.Promises` and `Backbone` style callbacks are definitely out.